package.json

```diff
@@ -85,6 +85,7 @@
     "gendocs": "typedoc --options typedoc.js src/ && node scripts/post-gendocs.mjs",
     "lint:eslint": "eslint 'src'",
     "lint:prettier": "prettier -c .",
+    "prettier": "prettier --write --list-different .",
```

Comment:
Lets to `format` to make the action clear.
```suggestion
    "format": "yarn lint:prettier -w",
```

Reply:
`--list-different` is helpful to print a list of formatted files, also I don’t like shortcuts flag, I always forget what they do

Reply:
Anyway it's your repo you can apply whatever you want

Reply:
Makes sense! My repo or not, I love the input - we're smarter together!

Lets do then
```suggestion
    "prettier": "yarn lint:prettier --write --list-different",
```